docs/images/devtools-golang-v1beta1.md

```diff
@@ -87,16 +90,15 @@ volumes:
 
 ### OCI image building
 
-!!! warning "Security Consideration"
-    OCI building requires that the devtools container run in priviliged mode. For
-    this to be more secure of a rootless OCI runtime (e.g. rootless dockerd) should
-    be used.
+!!! warning "Security Consideration" OCI building requires that the devtools
+    container run in priviliged mode. For this to be more secure of a rootless
+    OCI runtime (e.g. rootless dockerd) should be used.
```

Comment:
Revert to or it will break formatting. See: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title

```suggestion
!!! warning "Security Consideration"
    OCI building requires that the devtools container run in priviliged mode. For
    this to be more secure of a rootless OCI runtime (e.g. rootless dockerd) should
    be used.
```